packages/react-devtools-timeline/src/TimelineSearchContext.js

```diff
@@ -0,0 +1,166 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @flow
+ */
+
+import * as React from 'react';
+import {createContext, useMemo, useReducer} from 'react';
+
+import type {
+  ReactComponentMeasure,
+  ReactProfilerData,
+  ViewState,
+} from './types';
+
+type State = {|
+  profilerData: ReactProfilerData,
+  searchIndex: number,
```

Comment:
is this the currently selected result? if so might be useful to rename it to reflect that

Reply:
Right, right.

This is part of the search-state, e.g. text, results, index (within the results) all tied together.

I guess I could name it something more verbose (search results index?)